lib/blockchain/chaindb.js

```diff
@@ -1360,6 +1360,10 @@ class ChainDB {
     if (!Array.isArray(addrs))
       addrs = [addrs];
 
+    for (let addr of addrs)
```

Comment:
Are you sure this will work? I don't think the `addrs` array is being mutated here. Maybe this should go in the for loop below, right above `Address.getHash`. Then it also saves the need to iterate through the array twice.

Reply:
Fixed.